RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate REST API - getDepartmentInfo to Typescript

## Proposed changes

Migrate REST API - getDepartmentInfo to Typescript

## Issue(s)	


## How to test or reproduce

- Go to Omnichannel's room, inside RoomInfoView, check if appear the Department
![Screen Shot 2022-02-28 at 20 47 50](https://user-images.githubusercontent.com/47038980/156078002-7c2a309e-4986-4d7f-a5cf-eaa0c40d14a1.png)

- If doesn't appear the Department, forward the omnichannel's to a department and go to RoomInfoView again to see the department

## Screenshots

## Types of changes




## Checklist



## Further comments
